install.php

```diff
@@ -12,7 +12,7 @@
 	exit;
 }
 
-require_once __DIR__ . "/vendor/autoload.php";
+require_once "/vendor/autoload.php";
```

Comment:
I'll have to double check that for correctness, but have problems with elgg used as dependency, where autoloader is in vendor dir on root level, not relativley to `/vendor/elgg/elgg`. Try running Elgg using https://github.com/Srokap/elgg_as_dep, works reasonably well already, but things like icons don't work yet, due to mod dir being still kept in `/vendor/elgg/elgg`.